android/src/main/java/com/pdftron/reactnative/views/DocumentView.java

```diff
@@ -770,6 +770,10 @@ public void setCurrentToolbar(String toolbarTag) {
     }
 
     // Hygen Generated Props
+    public void setDisableReadingModeQuickMenu(boolean disabled) {
+        getToolManager().setReflowTextSelectionMenuEnabled(disabled);
```

Comment:
You need to inverse the logic
The flag is to disable the control

`setReflowTextSelectionMenuEnabled(true)` means the menu is ENABLED
`setReflowTextSelectionMenuEnabled(false)` means the menu is DISABLED

Reply:
i inversed the logic and renamed the prop setEnabledReadingModeQuickMenu - makes it more consistent to the native api that was added on android